gradio/queue.py

```diff
@@ -65,12 +83,15 @@ def __init__(
         self.queue_duration = 1
         self.live_updates = live_updates
         self.sleep_when_free = 0.05
+        self.progress_update_sleep_when_free = 0.2
```

Comment:
This sets the floor on how frequently the progress bar can be udated, right? Maybe 0.1 seconds is good?

Reply:
set to 0.1 for now.